backend/public-graph/graph/resolver.go

```diff
@@ -1628,15 +1628,16 @@ func (r *Resolver) sendErrorAlert(ctx context.Context, projectID int, sessionObj
 			numAlerts := int64(-1)
 			if err := r.DB.Raw(`
 				SELECT COUNT(*)
-				FROM alert_events
+				FROM error_alert_events ev
+				INNER JOIN error_objects obj
+				ON obj.id = ev.error_object_id
 				WHERE
-					project_id=?
-					AND type=?
-					AND (error_group_id IS NOT NULL
-						AND error_group_id=?)
-					AND alert_id=?
-					AND created_at > NOW() - ? * (INTERVAL '1 SECOND')
-			`, projectID, model.AlertType.ERROR, group.ID, errorAlert.ID, errorAlert.Frequency).Scan(&numAlerts).Error; err != nil {
+					obj.project_id=?
```

Comment:
After repeating this some time, I'm curious if we can do one of the following:
1. Remove the need to filter by `project_id` in these queries - seems redundant, but let me know if there is a security concern
2. If not, add project_id to the different types of alert event tables like it was on the `AlertEvent` table

Either one would save a join for _LogAlertEvents_ and _SessionAlertEvents_ but still would still need to join for _ErrorAlertEvents_ in order to filter by `error_group_id`

Reply:
don't think that the project_id is needed since the alert id should always be unique.